src/sections/Community/Handbook/community.js

```diff
@@ -3,29 +3,239 @@ import { Container, Row, Col } from "../../../reusecore/Layout";
-            <h2>Why contribute?</h2>
-            <p>Contributing to Open Source is an excellent way for gaining knowledge, engaging with the community, enhancing your skill set, and networking with different people across the world. As a contributor, working on a project like Meshery expands your experience with Docker, Kubernetes, Istio, Linkerd, Envoy, Octarine, App Mesh, Consul, Kuma, Maesh, and so on. With other amazing projects, you will also get to work with experienced software engineers in languages the projects use like Golang, Rust, Javascript (React, Gatsby, Nextjs, BillboardJs). In addition, contributors can get to work with DevOps tooling, continuous integration pipelining, documentation, and community management.</p>
+            <h2 id="communityGuidelines">Community Guidelines</h2>
+            <p>
+              Welcome to the Layer5 community! We’re happy to have you here and handhold you for your first contribution. Here are some tips to help you get started:
+            </p>
+            <p>
+              <img className="logo" src={Point} />
+              <strong>For Easy Identification -</strong> When setting up your slack workspace, your display name should be the same as your GitHub username for easy identification,
+              engage freely, and in whatever form you will: as a listener, user, contributor, maintainer, ambassador, or bystander. Involvement in any way is warmly received.
+            </p>
+            <p>
+              <img className="logo" src={Point} />
+              <strong>Familiarize with the community -</strong> As you get to familiarize yourself with the community, know that there are multiple initiatives for you to explore.
+              Some initiatives have working groups that meet at different times each week, while others use the weekly community meeting as their place of discussion. Details of each can be found on the <a href="https://bit.ly/2SbrRhe">community calendar</a>.
+              Everyone is welcome to join the calls and engage with other community members over Slack, the mailing list, and so on.
+            </p>
+            <p>
+              <img className="logo" src={Point} />
+              <strong>Familiarize with the projects -</strong> The following is a sample of the many active, ongoing community projects. One way of doing so is to review the Layer5 Repository Overview section of this handbook to get a better understanding of each project.
+              This document will help you to understand the projects based on their technology domain(s). Be sure to star the community’s <a href="https://github.com/layer5io">repositories</a> on GitHub!
```

Comment:
Here as well, could you please use the <Link> tag to redirect this to the repository overview page in the handbook itself. This github link is only for the layer5io github org.